src/Android/Xamarin.Android/Samples/Network Analysis/ClosestFacility/ClosestFacility.cs

```diff
@@ -0,0 +1,207 @@
+// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Android.App;
+using Android.OS;
+using Android.Widget;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.Tasks.NetworkAnalysis;
+using Esri.ArcGISRuntime.UI;
+using Esri.ArcGISRuntime.UI.Controls;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ArcGISRuntime.Samples.ClosestFacility
+{
+    [Activity(Label = "ClosestFacility")]
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Closest facility (interactive)",
+        "Network Analysis",
+        "Demonstrates how to solve a Closest Facility Task to find the closest route between a facility (hospital) and a incident (black cross). Tap to find the route to the nearest hospital.",
+        "")]
+    public partial class ClosestFacility : Activity
+    {
+        // Map view that will be used
+        private MapView _myMapView = new MapView();
+
+        // Holds locations of hospitals around San Diego
+        private List<Facility> _facilities;
+
+        // Graphics overlays for facilities and incidents
+        private GraphicsOverlay _facilityGraphicsOverlay;
+
+        // Symbol for facilities
+        private PictureMarkerSymbol _facilitySymbol;
+
+        private GraphicsOverlay _incidentGraphicsOverlay;
+
+        // Black cross where user clicked
+        private MapPoint _incidentPoint;
+
+        // Symbol for the incident
+        private SimpleMarkerSymbol _incidentSymbol;
+
+        // Used to display route between incident and facility to mapview
+        private SimpleLineSymbol _routeSymbol;
+
+        // Solves task to find closest route between an incident and a facility
+        private ClosestFacilityTask _task;
+
+        protected override void OnCreate(Bundle bundle)
+        {
+            base.OnCreate(bundle);
+
+            Title = "Closest Facility";
```

Comment:
Another minor nitpick: We use sentence case for titles in Runtime, so change this to 'Closest facility'.